ui/tests/integration/components/features-selection-test.js

```diff
@@ -0,0 +1,58 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { create } from 'ember-cli-page-object';
+import featuresSelection from 'vault/tests/pages/components/wizard/features-selection';
+import hbs from 'htmlbars-inline-precompile';
+import Service from '@ember/service';
+
+const component = create(featuresSelection);
+
+const permissionsService = Service.extend({
+  hasPermission(path) {
+    // This enables the Secrets and Authentication wizard items and disables the others.
+    const allowedPaths = ['sys/mounts/example', 'sys/auth', 'sys/auth/foo', 'sys/wrapping/wrap'];
+    if (allowedPaths.includes(path)) {
+      return true;
+    }
+    return false;
+  },
+});
+
+module('Integration | Component | features-selection', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    component.setContext(this);
```

Comment:
The new style tests don't need `setContext` or the `removeContext` calls below (I found this out when trying to take out jQuery, but that branch is still a WIP which is why it's still in a bunch of the files).